test/mobile/features/fixtures/nativeplugin/android/build.gradle

```
@@ -0,0 +1,25 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'
+    }
+}
+
+apply plugin: 'com.android.library'
+
+repositories {
+    google()
+    jcenter()
```

```suggestion
    mavenCentral()
    jcenter()
```

jcenter is deprecated so `mavenCentral` needs to come first in all repository blocks, so that it is used first

---

Added.